collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -44,6 +44,7 @@
     private static final String KEY_SHOW_SPLASH_SCREEN          = "show_splash_screen";
     private static final String KEY_DELETE_AFTER_SEND           = "delete_after_send";
     private static final String KEY_INSTANCE_FORM_SYNC          = "instance_form_sync";
+    private static final String KEY_LANGUAGE                    = "language";
```

Comment:
There's already an admin preference for turning on and off the ability to set language at the form level. It might be more helpful and less likely to cause conflicts to call this something like "app_language" / KEY_APP_LANGUAGE